### PR TITLE
Bugfix/iia 1045 cosmetic defects for pattern navigator UI

### PR DESCRIPTION
**Pull request by swaroopsalvi**

### CODE REVIEW

**Review by carldea — APPROVED** [review]
Looks good to me.

File: kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css [hunk 1]
```diff
@@ -2950,12 +2959,38 @@ Timeline Year line segment (will overlay change circles)
...
 }
 
+.pattern-instance-hbox .pattern-instance {
+    -fx-min-width: 260px;
+    -fx-pref-width: 260px;
+    -fx-max-width: 260px;
+
+}
+.pattern-instance-hbox .pattern-instance-hover-icon {
+         visibility: hidden;
```
Comment (dukke) [inline kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css:2977]: small formatting issue: too much extra spaced before "visibility". It's also present below.

**Comment by dukke** [comment]
Like @dholubek rightly suggested to me on some other issue would it be possible to attach a screenshot to this PR since this is a "visual issue"?

One other thing I noticed when I was talking to you yesterday in the video chat: the bumpout was not filling the whole height again, so I wonder if you need to pull and merge latest changes...

**Comment by swaroopsalvi** [comment]
@dukke and @dholubek , Will fix the space and the screenshot issue in next PR. Sorry did not see the comments when I hit merge.

**Comment by dukke** [comment]
@swaroopsalvi I just pulled and like I thought the concept bumpout is broken again (not filling height).

**Pull request merged**